Makefile

```diff
@@ -165,22 +176,22 @@ update-api-doc:
 
 # Build the docker image
 docker-build-dataset-controller: generate gen-openapi fmt vet
-	docker build --no-cache . -f docker/Dockerfile.dataset -t ${DATASET_CONTROLLER_IMG}:${GIT_VERSION}
+	docker build --no-cache --build-arg TARGETARCH=${ARCH} . -f docker/Dockerfile.dataset -t ${DATASET_CONTROLLER_IMG}:${GIT_VERSION}
```

Comment:
Can you set default value `amd64` for `ARCH` in front ?

Reply:
Sure, added.